.hermes/plans/workspace-dependency-cleanup-plan.md

```
@@ -0,0 +1,66 @@
+# Plan: Centralize workspace crate versions
+
+## Goal
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)